package.json

```diff
@@ -35,21 +35,19 @@
     "test": "CI=true yarn run test:jest",
     "test:jest": "react-app-rewired test --env=jsdom",
     "eject": "react-scripts eject",
-    "lint:flow": "flow",
     "lint:less": "stylelint 'src/**/*.less' --config .stylelintrc",
     "lint:eslint": "eslint -c node_modules/eslint-config-react-app/index.js --max-warnings 0 'src/**/*.js'",
     "lint:eslintfix": "eslint -c node_modules/eslint-config-react-app/index.js --max-warnings 0 --fix 'src/**/*.js'",
-    "lint:prettier-eslint": "prettier-eslint --list-different 'src/**/*.js' | ./lint_problems.sh",
-    "lint:prettier-eslintfix": "prettier-eslint --write 'src/**/*.js'",
-    "lint": "yarn run lint:flow && yarn run lint:prettier-eslint && yarn run lint:eslint && yarn run lint:less"
+    "lint:prettier-eslint": "prettier-eslint --list-different 'src/**/*.js' | ./lint_problems.sh && yarn run lint:eslint",
```

Comment:
This is causing eslint to run twice, since `yarn lint` calls `lint:prettier-eslint` followed by `lint:eslint`. We should avoid chaining scripts like this so there are no side effects. (Same comment applies to `lint:prettier-eslintfix`)

Reply:
(`&& yarn run lint:eslint` is the issue here)